assets/app/view/game/abilities.rb

```diff
@@ -85,6 +89,18 @@ def render_actions
       def actions_for(company)
         @game.round.actions_for(company)
       end
+
+      def render_sell_company_button
+        sell = lambda do
+          process_action(Engine::Action::SellCompany.new(
```

Comment:
can other games do this? it seems like the action already exists

looks like 1860 allows this already, can you do it in the same way 1860 does it?

Reply:
In 1860 the player can sell the company. In 18cz the corporation can sell the company after it was bought from the player.

Reply:
And another reason is that in 18CZ it has to be possible at every possible moment, because the money could be needed  to buy a new train etc.